tests/allman/dip1009.d.ref

```diff
@@ -0,0 +1,21 @@
+int foo(int arg)
+in
+{
+    assert(arg > 0);
+}
+out (result)
+{
+    assert(result == 0);
+}
+do
+{
+    return 0;
+}
+
+int bar(int arg)
+in(arg > 0)
+out(; true);
```

Comment:
I missed that: trailing `;` is invalid.

Reply:
i don't know how the tests site can work with this error

Reply:
As far as I understand the tests for dfmt concentrate on the formatted output, and not on the correctness of the input; putting this in libdparse's tests does make its test suite fail